### ISSUE TITLE
evac+fds

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#2403]
```
Please complete the following lines...

FDS Version:6.2.0
SVN Revision Number:22343
Compile Date:11.4.15
Smokeview Version/Revision:6.2.2/22325
Operating System:win 64 bit server

Describe details of the issue below:

attached are 3 small sample files that run when evac is excluded and don't run when
it is included.
1. evac+large fire- won't run
2. evac+samll fire+hvac- won't run
3. evac+small fire- does run but does not include the setup properites needed [hvac
and larger fire]

can you check on it please

thank you
Orit
```

Original issue reported on code.google.com by `orit13a` on 2015-05-19 09:32:35

<hr>
- _Attachment: [evac only fire.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-2445/comment-0/evac only fire.fds)_
- _Attachment: [evac big fire.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-2445/comment-0/evac big fire.fds)_
- _Attachment: [evacfire+hvac.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-2445/comment-0/evacfire+hvac.fds)_

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `mcgratta` on 2015-05-23 16:22:56

**Comment by gforney** [comment]
```
I'll chekc this.

Timo

```

Original issue reported on code.google.com by `tkorhon1` on 2015-05-25 08:15:44

**Comment by gforney** [comment]
```
I appreciate it very much
בתאריך 25 במאי 2015 11:15, <fds-smv@googlecode.com> כתב:
```

Original issue reported on code.google.com by `orit13a` on 2015-05-25 08:23:06

**Comment by gforney** [comment]
```
Well,

It might be a memory problem. I have now less points in
the y and z directions (dx=dy=dz=1.0m):

&MESH ID='Mesh03', IJK=360,64,10, XB=-1420.0,-1060.0,0.0,64.0,-0.25,9.75/

This seems to run nicely (the only_fire and big_fire). I did
also some other changes to speed up the debugging, see below.
I did not check the hvac version.

You should devide the mesh to many meshes. Now you have two parallel
tubes in the geometry. Make a separate mesh for each tube. And some
additional meshes for the connections between the tubes, if necessary
in the fire calculation. Evacuation mesh can also be devide, but 
this might not be necessary (not so many grid cells as in the fire 
meshes).


&TIME T_END=20.0, DT=0.5/

&DUMP DT_RESTART=25000.0,
      DT_HRR=1.0,
      DT_PART=1.0,
      SMOKE3D=.FALSE./

&MISC GVEC=-0.6295,0.0,-9.81, VISIBILITY_FACTOR=10.0,
      EVACUATION_DRILL=.FALSE., NO_EVACUATION=.FALSE./

 RADI TIME_STEP_INCREMENT=2/
&RADI RADIATION=.FALSE./

 Timo: These timers are taken away, they are not in any mesh. And they
 Timo: are also defined already above (same IDs).
 DEVC ID='TIMER5', QUANTITY='TIME', XYZ=0.0,0.0,0.0, SETPOINT=135.0/
 DEVC ID='TIMER2', QUANTITY='TIME', XYZ=0.0,0.0,0.0, SETPOINT=120.0/
 DEVC ID='TIMER22', QUANTITY='TIME', XYZ=0.0,0.0,0.0, SETPOINT=121.0/
 DEVC ID='TIMER6', QUANTITY='TIME', XYZ=0.0,0.0,0.0, SETPOINT=134.0/

Timo

```

Original issue reported on code.google.com by `tkorhon1` on 2015-05-25 08:50:42

<hr>
- _Attachment: [evac only fire.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-2445/comment-4/evac only fire.fds)_
- _Attachment: [evac_big_fire.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-2445/comment-4/evac_big_fire.fds)_

**Comment by tkorhon1** [comment]
Well, I'll close this issue.  The evac only fire.fds and evac big fire.fds and
evac_fire_hvac.fds seem to run "nicely" with the latest source code and
on our 64bit Linux cluster head node that has plenty of memory. I did
test just the "serial" version, i.e., did not use MPI, i.e., just one thread.

All input files have both evacuation and fire meshes present so they
are fire+evacuation calculations.

Note: I did run the cases about to the time = 25 s, then I stopped the runs.

Current Date     : October  6, 2016  12:32:50
Version          : FDS 6.5.2
Revision         : FDS0-1488-g43cac92
Revision Date    : Tue Oct 4 15:21:41 2016 -0400
Compilation Date : Oct 06, 2016  12:05:26

TimoK
